plugin/PDF/PdfDownload.php

```diff
@@ -65,4 +73,15 @@ public function download(PdfDocumentModel $pModel)
 		$url = $this->_pPdfDocumentFetcher->fetchUrl($pModelValidated);
 		$this->_pPdfDocumentFetcher->proxyResult($pModelValidated, $url);
 	}
+
+    /**
+     * @param bool $accept
+     */
+    public function settingGoogleBotAcceptIndex(bool $accept = true)
+    {
+        if ($accept === false)
+        {
+            $this->_httpHeadersGeneric->addHeader('X-Robots-Tag: googlebot: noindex, nofollow');
```

Comment:
This: 
```php
$this->_httpHeadersGeneric->addHeader('X-Robots-Tag: googlebot: noindex, nofollow');
```
should be part of the `download` method, That would make the usage more straightforward because that would disallow sequential coupling. In addition to that, it would save us from the boolean parameter.

Reply:
I updated